sdk/ml/azure-ai-ml/azure/ai/ml/_internal/entities/_additional_includes.py

```diff
@@ -46,5 +57,5 @@ def is_artifact_includes(self):
                     isinstance(additional_includes_configs, dict)
                     and self.get_config_key() in additional_includes_configs
                 )
-        except Exception:  # pylint: disable=broad-except
+        except (FileNotFoundError, yaml.YAMLError):
```

Comment:
Is it possible for any other exception here? If so, the exception will be raised directly, is it?

Reply:
Updated